src/pages/seyeonPark/Main/MainComponent/Feed.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import Comment from '../MainComponent/Comment';
+
+class Feed extends React.Component {
+  render() {
+    const {
+      commentBox,
+      comment,
+      getInputValue,
+      enterEvent,
+      uploadComment,
+      isHeartChange,
+      changeHeartColor,
+    } = this.props;
+    return (
+      <section className="feed">
+        <header className="feedHeader">
+          <div className="feedHeaderIdImg">
+            <img
+              alt="feedHeaderIdImg"
+              src="../../images/seyeonPark/feedidimg.jpeg"
```

Comment:
image source 들은 public 폴더에서 관리해주세요! 
그리고 public 폴더는 절대경로로 바로 접근할 수 있습니다
```suggestion
              src="/images/seyeonPark/feedidimg.jpeg"
```

Reply:
네 감사합니다. 적용하도록 하겠습니다.